README.md

```diff
@@ -27,19 +27,40 @@ We encourage others to contribute configurations they find useful. There is an [
 
 `%%gremlin` - Executes a Gremlin query against your database using web sockets. The results are similar to what the Gremlin console would return.
 
+`%%graph_notebook_config` - Sets the executing notebook's configuration to the JSON payload provided in the cell body.
+
+`%%graph_notebook_vis_options` - Sets the executing notebook's vis.js options to the JSON payload provided in the cell body.
```

Comment:
What vis.js options are available? So that we can describe the command more specifically.